packages/main/test/specs/MultiInput.spec.js

```diff
@@ -122,6 +122,27 @@ describe("MultiInput general interaction", () => {
 		assert.ok(await allTokens[5].getProperty("overflows"), `Token 5 should not overflow`);
 	});
 
+	it("adds a token after selection change", async () => {
```

Comment:
I would suggest to name the test as follows: "Should create a token on change event"